### PR TITLE
Make path correction

### PR DESCRIPTION
**Pull request by landml**
@tian - In the app "Batch Import from Staging", I put the slash at the end and it belonged at the beginning.

### CODE REVIEW

File: lib/kb_uploadmethods/Utils/BatchUtil.py [hunk 1]
```diff
@@ -286,11 +286,16 @@ def batch_import_assemblies_from_staging(self, params):
         staging_subdir = params.get('staging_subdir')
         workspace_name = params.get('workspace_name')
         assembly_set_name = params.get('assembly_set_name')
+        print ("STAGING BEFORE=", staging_subdir)
 
-        if staging_subdir.startswith(self.user_id + '/' ):
+        if staging_subdir.startswith('/' + self.user_id + '/' ):
             temp = len(self.user_id) + 1
```
Comment (Tianhao-Gu) [inline lib/kb_uploadmethods/Utils/BatchUtil.py]: remove the `temp` variable.

File: lib/kb_uploadmethods/Utils/BatchUtil.py [hunk 2]
```diff
@@ -286,11 +286,16 @@ def batch_import_assemblies_from_staging(self, params):
...
-            staging_subdir = staging_subdir[temp:]
-        
+            staging_subdir = staging_subdir.replace('/' + self.user_id + '/' , '/') 
+        elif staging_subdir.startswith('/' + self.user_id ):
+            temp = len(self.user_id) + 1
```
Comment (Tianhao-Gu) [inline lib/kb_uploadmethods/Utils/BatchUtil.py]: remove the `temp` variable.

File: lib/kb_uploadmethods/Utils/BatchUtil.py [hunk 3]
```diff
@@ -286,11 +286,16 @@ def batch_import_assemblies_from_staging(self, params):
...
         workspace_name = params.get('workspace_name')
         assembly_set_name = params.get('assembly_set_name')
+        print ("STAGING BEFORE=", staging_subdir)
 
-        if staging_subdir.startswith(self.user_id + '/' ):
+        if staging_subdir.startswith('/' + self.user_id + '/' ):
             temp = len(self.user_id) + 1
-            staging_subdir = staging_subdir[temp:]
-        
+            staging_subdir = staging_subdir.replace('/' + self.user_id + '/' , '/')
```
Comment (Tianhao-Gu) [inline lib/kb_uploadmethods/Utils/BatchUtil.py]: limit replace to 1 occurrence. In case `staging_subdir` has multiple user_id in it.

**Comment by landml** [comment]
Thank you!!

**Pull request merged**